lib/console_web/controllers/data_credit_controller.ex

```diff
@@ -326,7 +326,10 @@ defmodule ConsoleWeb.DataCreditController do
           end
 
         conn |> send_resp(:ok, Poison.encode!(%{ price: current_price["data"]["price"], next_price_timestamp: next_price_timestamp }))
-    end
+    else {:error, _} ->
+      conn
+      |> send_resp(404, "")
```

Comment:
i think this should send a 502, bad gateway

Reply:
502 Bad Gateway
The server was acting as a gateway or proxy and received an invalid response from the upstream server.

Reply:
oopsie you right

Reply:
addressed